src/components/tickets/index.tsx

```diff
@@ -0,0 +1,159 @@
+/* eslint-disable no-console */
+import CustomToolbar from './customToolbar';
+import { ITicketState } from '../../types/tickets';
+import { MenuCell } from './cells/menuCell';
+import TicketService from '../../service/TicketService';
+import TicketsModal from '../modals/tickets';
+import { tickets } from '../../store/tickets/selector';
+import { useAppDispatch } from '../../store/hooks';
+import { useSearchParams } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import { StyledBox, StyledDataGrid } from './styled';
+import { columns, pageSizeOptions } from './helper';
+import { deleteTicket, setTicketState } from '../../store/tickets/slice';
+import { useCallback, useEffect, useState } from 'react';
+
+const TicketsTable = () => {
+  const [isModalOpen, setIsModalOpen] = useState(false);
```

Comment:
will take a look at the all of this stuff

Reply:
First: 
We don't use `[isModalOpen, setIsModalOpen]` and `toggleModal` in this component

Reply:
fix: moved to customToolbar